custom/mlservermllib/models/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .mllib import MLlibModel
+
+__all__ = ["MLlibModel"]
```

Comment:
The `models` package can probably act as the top-level package for `mlserver_mllib`